CHANGELOG.md

```diff
@@ -21,8 +21,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 _ `_optional` subpackage for managing optional dependencies
 - Acquisition function for active learning: `qNIPV`
 - Abstract `ContinuousNonlinearConstraint` class
-- `ContinuousCardinalityConstraint` class and corresponding uniform sampling mechanism
+- Abstract `CardinalityConstraint` class and
```

Comment:
@Waschenbacher it seems like this PR is based on top of #270 , right? So please ping me here once that is merged and this is in general ready for review

Reply:
Yes, good guess. I have updated the explanation above. Thanks, will ping you and looking forward to your review 👍